source/jormungandr/jormungandr/scenarios/helper_classes/fallback_durations.py

```diff
@@ -148,23 +170,27 @@ def _do_request(self):
         places_isochrone = [p for p in proximities_by_crowfly if p.uri not in all_free_access]
 
         result = {}
+        orig_to_parking_durations = {}
         # Since we have already places that have free access, we add them into the result
         [result.update({uri: DurationElement(0, response_pb2.reached)}) for uri in all_free_access]
 
         # There are two cases that places_isochrone maybe empty:
         # 1. The duration of direct_path is very small that we cannot find any proximities by crowfly
         # 2. All proximities by crowfly are free access
         if not places_isochrone:
-            return result
+            return result, orig_to_parking_durations
```

Comment:
for me, the `orig_to_parking_durations` is not necessary, as you already have that information in the returned `proximities_by_crowfly`

Reply:
But, **proximity_by_crowfly** is not returned, this is an _internal function_. 
If we want to handle **origin to parking informations**, we need to return it.